crates/aptos-dkg/src/range_proofs/mod.rs

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod univariate_range_proof;
```

Comment:
A better name would be `dekart_univariate`. You should not need the `_range_proof` qualifer, since this is already scoped in `range_proofs`.